<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sharePage</class>
 <widget class="QWidget" name="sharePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>772</height>
   </rect>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>761</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>949</width>
      <height>759</height>
     </rect>
    </property>
    <widget class="QLabel" name="shotByLabel_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>100</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Jakopäivä</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="shareSharedPortionsTableWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>380</y>
       <width>420</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kaadot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shareSavePushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tallenna</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="shareKillsTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>920</width>
       <height>192</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Valitse kaato, jonka lihoja jaetaan</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="shareDateEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="Finnish" country="Finland"/>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2022</year>
       <month>11</month>
       <day>15</day>
      </date>
     </property>
    </widget>
    <widget class="QComboBox" name="memberComboBox">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>300</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QWebEngineView" name="shareSankeyWebEngineView" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>500</width>
       <height>331</height>
      </rect>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_13">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Jäsen</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ruhon osa</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portionComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_15">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>330</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jaetut ruhon osat</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>280</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="chosenItemLabel">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>0</y>
        <width>260</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ei valittua kaatoa</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="shotByLabel_14">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Seurue</string>
     </property>
    </widget>
    <widget class="QComboBox" name="partyComboBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>300</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="sortSharesComboBox">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>350</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_16">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>330</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lajittele</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortKillsComboBox">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_17">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>0</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lajittele</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shareEditPushButton">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>250</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Muokkaa jakoja...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shareRemovePushButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>250</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Poista jakoja...</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diagrammi</string>
     </property>
    </widget>
    <widget class="QComboBox" name="shareSankeyComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
